packages.windows/spartan/buildinfo.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
The spartan package was removed a few months ago, see https://github.com/dcos/dcos/pull/1672.